spec/models/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe '#update_post_comments_counter' do
+    it 'updates the post comments_counter attribute' do
+      user = User.create(name: 'Tom')
+      post = Post.create(title: 'Hello', author: user)
+      comment = Comment.create(author: user, post:)
+
+      comment.update_post_comments_counter
+
+      expect(post.reload.comments_counter).to eq(1)
+    end
+  end
+end
```

Comment:
- [ ] Great job so far. However, none of these tests seem not to pass. Kindly look into them and fix.

### Test Result
![comment](https://github.com/adarachel/rails-blog/assets/26597432/a808b49c-3774-48cd-93a3-6a777e096767)